README.md

```diff
@@ -91,6 +98,19 @@ After executing `poetry install`:
 poetry publish
 ```
 
+#### Building Docker images from source
+
+```bash
+docker build . -t 23jura23/hermes_se2021:your_tag
```

Comment:
Why `23jura23`?)

Reply:
I should've published it on Docker Hub and had only my personal account, so I tagged it and pushed there :)

Of course I could create the standalone Docker Hub account for our app, if it would be better